compute/instances.go

```diff
@@ -963,8 +974,7 @@ func (api *_Instance) toNative() (*Instance, error) {
 // submit an API call to the vmapi machine endpoint
 func (mcns *InstanceCNS) toTags(m map[string]interface{}) {
 	if mcns.Disable != nil {
-		s := fmt.Sprintf("%t", mcns.Disable)
-		m[CNSTagDisable] = &s
+		m[CNSTagDisable] = fmt.Sprintf("%t", *mcns.Disable)
```

Comment:
This should likely use `strconv.FormatBool` from the standard library instead of `fmt.Sprintf`